sound/soc/intel/common/soc-acpi-intel-mtl-match.c

```diff
@@ -143,6 +194,12 @@ struct snd_soc_acpi_mach snd_soc_acpi_intel_mtl_sdw_machines[] = {
 		.drv_name = "sof_sdw",
 		.sof_tplg_filename = "sof-mtl-rt715-rt711-rt1308-mono.tplg",
 	},
+	{
+		.link_mask = GENMASK(3, 0),
+		.links = mtl_3_in_1_sdca,
+		.drv_name = "sof_sdw",
+		.sof_tplg_filename = "sof-mtl-rt711-rt1316-rt714.tplg",
```

Comment:
@plbossart Due to the codec layout change, should we keep the similar name as used in tgl/cml (sof-mtl-rt711-rt1316-rt714.tplg), or use a new name like `sof-mtl-rt711-l0-rt1316-l2-l3-rt714-l1.tplg`, or write codec in link order `sof-mtl-rt711-rt714-rt1316.tplg`?

Reply:
> @plbossart Due to the codec layout change, should we keep the similar name as used in tgl/cml (sof-mtl-rt711-rt1316-rt714.tplg), or use a new name like `sof-mtl-rt711-l0-rt1316-l2-l3-rt714-l1.tplg`, or write codec in link order `sof-mtl-rt711-rt714-rt1316.tplg`?

Use `sof-mtl-rt711-l0-rt1316-l23-rt714-l1.tplg` please. We will very likely have `sof-mtl-rt711-l0-rt1316-l12-rt714-l3.tplg` for other products.